independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/CodestartProjectGeneration.java

```diff
@@ -17,6 +17,8 @@
 
 final class CodestartProjectGeneration {
 
+    private static final boolean IS_WINDOWS_OS = System.getProperty("os.name").toLowerCase().contains("win");
```

Comment:
What if the OS is "Darwin"?

There is a `smallrye-common-os` library that implements that easily